react/src/pages/utils.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { forwardRef, ReactElement, Ref } from "react";
+import { Slide } from "@material-ui/core";
+import { TransitionProps } from "@material-ui/core/transitions";
+
+export const SlideUpTransition = forwardRef((
```

Comment:
Not sure if this transition is necessary, because all the other pages use the default transition.

Reply:
Yeah, it's only used in the two modals on the analysis page, probably because of when they were written/copy-pasted.